### DISCUSSION TITLE
Will the V2 update add wheel support?

### DISCUSSION THREAD

**Discussion by SnkXIII** [nefarius/ViGEmBus#170] [Q&A] [answered]
I don't want to bog you with details, but my idle curiosity question does come from a kind of frustration with both LGS, GHUB and now thrustmaster.
There's no per-game support with thrustmaster stuff, and it seems like I can't use their target software to remap my wheel like I need to (they can support their HOTAS in this way but not wheel afaik)... I liked to map keyboard keys to my logitech wheel until it fully stopped allowing me on LGS or GHUB. Also unable to duplicate XInput buttons to other buttons on the wheel, like having LSB and RSB both mapped as LSB. Thrustmaster seems worst on all counts

There's pros to having something like vigem (or something using it like DS4W) that I will likely miss out because I'm quite new into the controller type overall but...

I can't find any sort of software for my wheel that'll let me remap buttons, play with axes readings (if your pedals ship reading inverted, maxed out to windows when not pressed down, it's a case of inverting the positive and negative wires on your pedals to fix and I've done that already, gladly, as it's shocking how unfixable this can be if the game exposes no options to you) 
and well vigem has been very necessary in supporting controllers but also helping to unlock functionality not otherwise there. If there could be some sort of wheel button remapper with force feedback support/passthrough (idk), that'd be amazing, and I think would be novel, I've tried to look for something to get me close to what I need, but got nowhere so far... Still looking

I can only guess at what difficulties would pop up, where they'd arise, in trying to give support to so many wheels out there... But hoping something can happen without too much difficulty to buff the QOL aspect to all of these wheels! I'd be glad to know how feasible this all is, and all the best for the future of the project and thanks for basically being the one to support all the other controllers I have!

**Comment by nefarius** [comment, marked as answer]
Yes, I do plan to emulate a racing wheel, other things take priority currently but it's on the list.

**Comment by SnkXIII** [comment]
That's brilliant to know, thank you so much! Will be excited to see what happens in future

**Reply by nefarius** [reply]
I own one fairly middle to high class Logitech wheel I'll use to model it after. Don't know the model name from memory but I can post it here later.

**Reply by SnkXIII** [reply]
ah nice! mine is a g920 and i moved up to thrustmaster t300rs gt! i was wondering if you mightve come across any software other than manufacturer provided, that i might have missed? I forgot about RS Mapper as I was writing this initial question, but only because I couldn't figure out how to use it properly, so it might be worth giving it more attempts here soon

**Reply by nefarius** [reply]
I have a [Logitech Driving Force GT](https://github.com/CircumSpector/Research/tree/master/Logitech%20Driving%20Force%20GT), IDK if that is good, mid-tier or bad but it got the job done for me 😅
